source/neuropod/backends/tensorflow/tf_backend.cc

```diff
@@ -250,15 +202,57 @@ void TensorflowNeuropodBackend::load_model_internal()
     auto                     config_stream = loader_->get_istream_for_file("0/config.json");
     setup_node_mapping_and_init_ops(*config_stream, node_name_mapping_, init_ops);
 
+    // Run init ops if any
+    for (const auto &op_name : init_ops)
+    {
+        check_tf_status(session_->Run({}, {}, {op_name}, nullptr));
+    }
+}
+
+void TensorflowNeuropodBackend::load_model_internal()
+{
+    // Load custom ops (if any)
+    for (const auto &item : model_config_->custom_ops)
+    {
+        const auto path = "0/ops/" + item;
+        const auto hash = loader_->get_hash_for_file(path);
+
+        // Don't load a custom op if we've already loaded it
+        std::lock_guard<std::mutex> lock(loaded_op_mutex);
+        if (loaded_op_hashes.count(hash) == 0)
```

Comment:
Can you comment on how this loaded_op_hashes is used?

Reply:
This is preexisting code that I just moved around (unrelated to savedmodel). Is it okay if I clarify in a follow up PR if necessary?

The functionality seems pretty clear to me based on the comments and the code in lines 214 to 239, but I've spent a lot of time in this codebase so I could think it's more clear than it actually is haha.

I'll follow up offline

Reply:
Correct!